bundles/core/src/test/resources/teaser/exporter-teaser9.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Teaser",
+  "description": "Description",
+  "linkURL": "http://www.adobe.com",
```

Comment:
would go with either `link` or `url` for this property rather than a combination and make it consistent with the output of the actions.

Reply:
We used the same property name as the `Image` component, it's needed if we want to delegate rendering.